src/components/Nav/Nav.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { useState } from "react";
 import * as s from "./Nav.style";
-import logoSrc from "@Assets/images/full-white-2.png";
```

Comment:
Have a look at the path resolution used by craco prefixed by @ and update your imports so they don't use a relative path